docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture designed by Arc that enables it to use the reusable contracts provided by Arc.  Each DAO has an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
+
+**The controller** is a central internal hub for all of the components of the DAO.
+
+**The token** is used as currency in operations like staking in `GenesisProtocol`, vesting agreements in `VestingScheme` and rewards in `ContributionReward`.
```

Comment:
> currency

does token is more appropriate word here ?

Reply:
> does token is more appropriate word here ?

That would create a circular definition (a token is a token).  "Currency" has a broad meaning of a mechanism for representing and transferring something of value.